### PR TITLE
Pm 2022 11 semantic html

### PR DESCRIPTION
**Pull request by phillimorland**
adding more semantic markup to help with sr

### CODE REVIEW

File: meinberlin/apps/contrib/templates/meinberlin_contrib/item_detail.html [hunk 1]
```diff
@@ -121,14 +119,14 @@ <h2 class="moderator-statement__title">{% trans 'Official Statement' %}</h2>
                         {{ object.moderator_statement.statement | safe }}
                     </div>
                 </div>
-            </div>
         {% endif %}
     {% endblock %}
```
Comment (fuzzylogic2000) [inline meinberlin/apps/contrib/templates/meinberlin_contrib/item_detail.html:123]: The endblock needs to be more indented to match the opening block tag. Maybe the whole block after the opening thing should be (never sure about the indentation with the block tags).

File: meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html [hunk 2]
```diff
@@ -199,33 +196,31 @@ <h2>{{ event.name }}</h2>
...
-        </div>
         {% else %} <!-- if just one module and no phase view to dispatch to -->
         {% include "a4modules/includes/module_detail_phase.html" %}
         {% endif %}
         <!-- these blocks are only filled when in phase view -->
-        <div class="container">
             <div class="offset-lg-3 col-lg-6">
                 {% block project_action %}{% endblock %}
             </div>
         </div>
```
Comment (fuzzylogic2000) [inline meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html:206]: The closing div tag also needs to go

File: meinberlin/templates/a4modules/includes/module_detail_phase.html [hunk 3]
```diff
@@ -1,58 +1,54 @@
...
-            <i class="fas fa-lock u-spacer-right" aria-hidden="true"></i>{% trans 'This project is not publicly visible. Only invited users can see content and actively participate.' %}
-        </div>
-    {% elif project.is_semipublic %}
-        <div class="">
-            <i class="fas fa-eye u-spacer-right" aria-hidden="true"></i>{% trans 'This project is publicly visible. Invited users can actively participate.' %}
-        </div>
-    {% endif %}
+<div class="offset-lg-3 col-lg-6 u-spacer-bottom-double">
+{% if project.is_private %}
+    <div class="">
```
Comment (fuzzylogic2000) [inline meinberlin/templates/a4modules/includes/module_detail_phase.html:46]: You didn't add it, but the empty class can be removed

File: meinberlin/templates/a4modules/includes/module_detail_phase.html [hunk 4]
```diff
@@ -1,58 +1,54 @@
...
+        <i class="fas fa-lock u-spacer-right" aria-hidden="true"></i>{% trans 'This project is not publicly visible. Only invited users can see content and actively participate.' %}
+    </div>
+{% elif project.is_semipublic %}
+    <div class="">
```
Comment (fuzzylogic2000) [inline meinberlin/templates/a4modules/includes/module_detail_phase.html:50]: Same here

**Review by fuzzylogic2000 — COMMENTED** [review]
Very nice! found total nit picks.

File: meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html (hunk 2 above)
Comment (phillimorland) [reply meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html:206]: @fuzzylogic2000 that's matching the tag at 175 as i just removed all the additional containers

**Review by fuzzylogic2000 — APPROVED** [review]
Cool!

**Pull request merged**